tests/test_coordUtils.py

```diff
@@ -0,0 +1,325 @@
+                fieldAngle = (xrad, yrad)
+                vector = coordUtils.fieldAngleToVector(fieldAngle, flipX)
+                self.assertAlmostEqual(np.linalg.norm(vector), 1)
+                if testOrientation:
+                    # orientation should match
+                    orientationFromFieldAngle = math.atan2(yrad, signx*xrad)*radians
+                    # field angle x = vector y, field angle y = vector z
+                    orientationFromVector = math.atan2(vector[2], vector[1])*radians
+                    self.assertAnglesAlmostEqual(orientationFromVector, orientationFromFieldAngle)
+
+                # now test as spherical geometry
+                sp = SpherePoint(Vector3d(*vector))
+                separation = sp00.separation(sp)
+                predictedSeparation = math.hypot(xrad, yrad)*radians
+                self.assertAnglesAlmostEqual(predictedSeparation, separation)
+                if testOrientation:
+                    bearing = sp00.bearingTo(sp)
+                    self.assertAnglesAlmostEqual(orientationFromFieldAngle, bearing)
+
+                # test round trip through vectorToFieldAngle
+                fieldAngleFromVector = coordUtils.vectorToFieldAngle(vector, flipX)
+                np.testing.assert_allclose(fieldAngleFromVector, fieldAngle, atol=1e-15)
+
+    def testVectorToFieldAngle(self):
+        # note: more sophisticated cases are tested by testFieldAngleToVector
+        for flipX in (False, True):
+            signx = -1 if flipX else 1
+            for magMultiplier in (0.001, 1, 1000):
+                for vector, predictedFieldAngleDeg in (
+                    ((1, 0, 0), (0, 0)),
+                    ((0, 1, 0), (signx*90, 0)),
+                    ((0, 0, 1), (0, 90)),
+                ):
+                    predictedFieldAngle = [val*RadPerDeg for val in predictedFieldAngleDeg]
+                    fieldAngle = coordUtils.vectorToFieldAngle(vector, flipX)
+                    np.testing.assert_allclose(predictedFieldAngle, fieldAngle, atol=1e-15)
+
+    def testComputeShiftedPlanePosZeroFieldAngle(self):
+        """Test computeShiftedPlanePos with zero field angle
+
+        This should result in no change
+        """
+        zeroFieldAngle = (0, 0)
+        for planePos in (
+            (-1000, -2000),
+            (0, 0),
+            (5000, 4000),
+        ):
+            for shift in (-500, 0, 500):
+                shiftedPlanePos = coordUtils.computeShiftedPlanePos(planePos, zeroFieldAngle, shift)
+                self.assertPairsAlmostEqual(planePos, shiftedPlanePos)
+
+    def testComputeShiftedPlanePosZeroShift(self):
+        """Test computeShiftedPlanePos with zero shift
+
+        This should result in no change
+        """
+        zeroShift = 0
+        for planePos in (
+            (-1000, -2000),
+            (0, 0),
+            (5000, 4000),
+        ):
+            for fieldAngle in (
+                (0.5, 0.5),
```

Comment:
I know you liked the idea of changing these values to not be the same, so just pointing out that this one has (0.5, 0.5) for its first pair in case you want to change that. Given the others differ I think this is highly unlikely to matter though.

Reply:
In this case I like having one at 45 degrees; other values are also tested